doc/setup.md

```diff
@@ -240,6 +263,30 @@ If the command outputs information about a service, it means that port 5000 is a
 6. Change users by navigating to `http://localhost:5000/become_user/<#>` where # is a number from 1-5.
    Different users have different privileges and can see more or less of the demo data.
 
+<a id="port-configuration"></a>
+## Port Configuration
+
+Skyportal uses two distinct ports in its configuration:
+
+1. Internal Port - Where the application is actually hosted
+2. Public-Facing Port - Where external users access the application
+
+If you need to use different ports (e.g., port 5000 is already in use) add the following changes to `skyportal/config.yaml.defaults` override the defaults:
```

Comment:
changes should be made to config.yaml (copied from the default), never to the default file itself!

Reply:
True! Oopsie. Thanks for catching this! 😄